client/src/main/scala/io/delta/sharing/spark/RemoteDeltaLog.scala

```diff
@@ -41,17 +41,19 @@ import io.delta.sharing.spark.perf.DeltaSharingLimitPushDown
 private[sharing] class RemoteDeltaLog(
```

Comment:
Consider adding comments on how the init metadata is used.

Reply:
added